vendor/github.com/codahale/hdrhistogram/.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: go
+go:
+  - 1.5
```

Comment:
Remove these I think. Just do 1.11 and 1.12 and maybe tip.

Reply:
nvm this is vendor, but maybe update the vendored stuff ;)

Reply:
hmm that repository seems archived 🤔

Reply:
Interesting.. For now ok I guess.